integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestNonTransactionalCarbonTable.scala

```diff
@@ -1460,8 +1459,13 @@ class TestNonTransactionalCarbonTable extends QueryTest with BeforeAndAfterAll {
   }
 
   test("Read sdk writer Avro output Array Type with Default value") {
-    buildAvroTestDataSingleFileArrayDefaultType()
-    assert(new File(writerPath).exists())
+    // avro1.8.x Parser donot handles default value , this willbe fixed in 1.9.x. So for now this
+    // will throw exception. After upgradation of Avro we can change this test case.
```

Comment:
This test case is required. Along with this can you upgrade to Avro 1.9.X also ? We cannot have it as a separate activity.

Reply:
community knows this issue , they said this will be fixed in 2.x version